src/graph.ts

```diff
@@ -87,9 +87,24 @@ function typeFromCfn(cfnType: string): string {
     return typeParts[2];
 }
 
+/**
+ * Get the 'type' from the FQN
+ *
+ * The fqn is automatically generated by JSII and is in the format `module.submodule.ClassName`
+ *
+ * For example,
+ *    import { Bucket } from 'aws-cdk-lib/aws-s3';
+ * `Bucket` would have the fqn of `aws-cdk-lib.aws-s3.Bucket`
+ * and we would return `aws-cdk-lib/aws-s3:Bucket`
+ *
+ * @param fqn - The fully qualified name of the construct
+ * @returns The pulumi resource type (i.e. aws-cdk-lib/aws-s3:Bucket)
```

Comment:
Out of scope from this PR, but wanted to highlight. What is this "aws-cdk-lib/aws-s3:Bucket the pulumi resource type"? What is the taxonomy here. This does not seem to be the provider type token I've seen before. In cases like this I have found it very helpful to have a wrapper or alias type instead of `string` for such a concept, so `jump-to-definition` can help clarify some of these questions and what is this, and accumulate doc comments.

Reply:
Ah so this is the type token of component resources in pulumi-cdk?

Reply:
Yes, but I think it's a good idea to add one of those alias types to make this easier to follow.

Reply:
We have a canonical type in the Go SDK for resource tokens but I'm not sure we do (yet) for Node.